go/fn/object_test.go

```diff
@@ -183,6 +183,35 @@ spec:
 	}
 }
 
+func TestIsLocalConfig(t *testing.T) {
+	functionConfig := []byte(`
+apiVersion: fn.kpt.dev/v1alpha1
+kind: SetLabels
+metadata:
+ name: my-config
+ annotations:
+   config.kubernetes.io/local-config: "true"
+labels:
+ color: orange
+ fruit: apple
+`)
+	item := []byte(`
+apiVersion: v1
+kind: Service
+metadata:
+ name: whatever
+ labels:
+   app: myApp
+`)
+	rl := &ResourceList{}
+	config, _ := ParseKubeObject(functionConfig)
+	itemObj, _ := ParseKubeObject(item)
+	rl.Items = []*KubeObject{config, itemObj}
```

Comment:
nit: i would suggest using a resource list as data input. and put both KRM resource as "items". 
Reason:
I know why you use functionConfig and item here, but this variable names can be misleading since the IsLocalConfig normally is used to select items, and that won't be applied to `functionConfig`. You can use Kptfile example instead.